src/simplicial-hott/03-simplicial-type-theory.rzk.md

```diff
@@ -66,12 +66,33 @@ Products:
 #def ∂□ : (2 * 2) -> TOPE
   := \ (t ,s) -> ((∂Δ¹ t) /\ (Δ¹ s)) \/ ((Δ¹ t) /\ (∂Δ¹ s))
 
--- the vertical boundary of the square 
+-- the vertical boundary of the square
 #def ∂Δ¹×Δ¹ : (2 * 2) -> TOPE
   := shapeProd 2 2 ∂Δ¹ Δ¹
 
+-- the horizontal boundary of the square
+#def Δ¹×∂Δ¹ : (2 * 2) -> TOPE
+  := shapeProd 2 2 Δ¹ ∂Δ¹
 
 -- the prism from a 2-simplex in an arrow type
 #def Δ²×Δ¹ : (2 * 2 * 2) -> TOPE
-  := shapeProd (2 * 2) 2 Δ² Δ¹  
-```  
+  := shapeProd (2 * 2) 2 Δ² Δ¹
+```
+
+Intersections:
+
+```rzk
+-- the conjunction of topes defines the intersection of shapes
+#def shapeIntersection
+  (I : CUBE) (ψ χ : I -> TOPE) : I -> TOPE
+  := \ t -> ψ t /\ χ t
```

Comment:
I think it might make sense to support `/\` for tope families directly in `rzk` so that `ψ /\ χ` automatically would mean `\ t -> ψ t /\ χ t` when appropriate.

Reply:
That could make sense; it would be nice to have a special notation for intersection at least. You're not planning on implementing something like [Agda's infix/mixfix syntax](https://agda.readthedocs.io/en/v2.6.3/language/mixfix-operators.html)? Then this definition could be made internally. 😄

Reply:
Or actually, you would also need general implicit arguments to get the appropriate internal definition.